pkg/koordlet/resmanager/cpu_suppress.go

```diff
@@ -316,10 +323,45 @@ func (r *CPUSuppress) adjustByCPUSet(cpusetQuantity *resource.Quantity, nodeCPUI
 			cpuIdToPool[int32(cpuID)] = apiext.GetPodQoSClass(podMeta.Pod)
 		}
 	}
+
+	topo := r.resmanager.statesInformer.GetNodeTopo()
+	cpuManagerPolicy := apiext.KubeletCPUManagerPolicy{}
+	if topo == nil {
+		return
+	}
+
+	if policy, ok := topo.Annotations[apiext.AnnotationKubeletCPUManagerPolicy]; ok {
+		if err := json.Unmarshal([]byte(policy), &cpuManagerPolicy); err != nil {
+			klog.Errorf("Failed to unmarshal cpuManagerPolicy by kubelet, err: %v.\n", err)
+			return
+		}
+	}
+	reserved, err := cpuset.Parse(cpuManagerPolicy.ReservedCPUs)
+	if err != nil {
+		klog.Errorf("Failed to parse reserved cpu by kubelet, err: %v.\n", err)
+		return
+	}
+
+	reservedByNode := cpuset.CPUSet{}
+	if cpus, ok := topo.Annotations[apiext.CPUsReservedByNode]; ok {
+		reservedByNode, err = cpuset.Parse(cpus)
+		if err != nil {
+			klog.Errorf("Failed to parse reserved cpu by nodetopo, err: %v.\n", err)
+			return
+		}
+	}
+
 	var lsrCpus []koordletutil.ProcessorInfo
 	var lsCpus []koordletutil.ProcessorInfo
 	// FIXME: be pods might be starved since lse pods can run out of all cpus
 	for _, processor := range nodeCPUInfo.ProcessorInfos {
+		cpuset := cpuset.NewCPUSet(int(processor.CPUID))
+		if cpuset.IsSubsetOf(reserved) {
```

Comment:
`reserved` are CPUs reserved by kubelet, which can be shared with besteffort, so we should not skip them.

Reply:
done